### PR TITLE
Update Go version

### PR DESCRIPTION
**Pull request by mchompalova**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,34 +1,30 @@
...
+  && echo "Build flags: $LDFLAGS" 
 
-WORKDIR /
+# Multi-stage build - copy certs and the binary into the image
+FROM scratch
+WORKDIR /	WORKDIR /
+COPY --from=0 /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=0 /artifacts/* /
 
 CMD [ "/startup.sh" ]
```
Comment (ivan-p-nikolov) [inline Dockerfile]: We don't copy `startup.sh`, `brandMappings.json` or `config.json` in the scratch build.

File: Dockerfile [hunk 2]
```diff
@@ -1,34 +1,30 @@
...
-  && apk del .build-dependencies \
-  && rm -rf $GOPATH /var/cache/apk/*
+  && LDFLAGS="-s -w -X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+  && CGO_ENABLED=0 go build -a -o /artifacts/${PROJECT} -ldflags="${LDFLAGS}" \
+  && echo "Build flags: $LDFLAGS" 
 
-WORKDIR /
+# Multi-stage build - copy certs and the binary into the image
+FROM scratch
+WORKDIR /	WORKDIR /
```
Comment (ivan-p-nikolov) [inline Dockerfile]: I'm not sure what the second `WORKDIR /` is suppose to do.

File: Dockerfile [hunk 3]
```diff
@@ -1,34 +1,30 @@
-FROM golang:1.8-alpine
+FROM golang:1
 
 ENV PROJECT=publish-availability-monitor
-COPY . /${PROJECT}-sources/
+
+ENV ORG_PATH="github.com/Financial-Times"
+ENV BUILDINFO_PACKAGE="${ORG_PATH}/${PROJECT}/vendor/${ORG_PATH}/service-status-go/buildinfo."
```
Comment (ivan-p-nikolov) [inline Dockerfile]: Build info would not work with that path.

File: Dockerfile [hunk 4]
```diff
@@ -1,34 +1,30 @@
...
-  && echo "Build flags: $LDFLAGS" \
-  && echo "Fetching dependencies..." \
-  && go get -u github.com/kardianos/govendor \
-  && $GOPATH/bin/govendor sync \
-  && go build -ldflags="${LDFLAGS}" \
-  && mv ${PROJECT} /${PROJECT} \
-  && apk del .build-dependencies \
-  && rm -rf $GOPATH /var/cache/apk/*
+  && LDFLAGS="-s -w -X '"${BUILDINFO_PACKAGE}$VERSION"' -X '"${BUILDINFO_PACKAGE}$DATETIME"' -X '"${BUILDINFO_PACKAGE}$REPOSITORY"' -X '"${BUILDINFO_PACKAGE}$REVISION"' -X '"${BUILDINFO_PACKAGE}$BUILDER"'" \
+  && CGO_ENABLED=0 go build -a -o /artifacts/${PROJECT} -ldflags="${LDFLAGS}" \
```
Comment (ivan-p-nikolov) [inline Dockerfile]: add `-mod=readonly` to the build

File: .circleci/config.yml [hunk 5]
```diff
@@ -13,35 +13,33 @@ jobs:
       - run:
           name: External Dependencies
           command: |
-            go get -u github.com/mattn/goveralls
-            go get -u github.com/jstemmer/go-junit-report
-            go get -u github.com/kardianos/govendor
-            go get -u github.com/haya14busa/goverage
+            GO111MODULE=off go get github.com/mattn/goveralls
```
Comment (ivan-p-nikolov) [inline .circleci/config.yml]: why remove the update flag from this `get`

File: Dockerfile (hunk 3 above)
Comment (mchompalova) [reply Dockerfile]: fixed

File: Dockerfile (hunk 1 above)
Comment (mchompalova) [reply Dockerfile]: Done.

**Pull request merged**